tests/gtest/avifmetadatatest.cc

```diff
@@ -284,6 +286,22 @@ TEST(MetadataTest, ExifOrientation) {
   EXPECT_EQ(image->width, temp_image->width /* should be height here */);
 }
 
+TEST(MetadataTest, AllExifOrientations) {
+  const ImagePtr image =
+      testutil::ReadImage(data_path, "paris_exif_orientation_5.jpg");
+  ASSERT_NE(image, nullptr);
+  image->transformFlags = AVIF_TRANSFORM_NONE;
```

Comment:
Optional: It seems better to move these four lines (290-293) into the for loop so that each iteration of the for loop starts from a clean image. Right now each iteration starts from the results of the previous iteration.

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/2735